nodejs/client.js

```diff
@@ -1,46 +1,35 @@
-var async = require('async');
-var Client = require('hazelcast-client').Client;
+'use strict';
 
-var clientConfig = {
+const { Client } = require('hazelcast-client');
+
+const clientConfig = {
     network: {
         clusterMembers: [
             'hz-hazelcast'
         ]
     }
-}
-
-Client.newHazelcastClient(clientConfig).then(function (hazelcastClient) {
-    var client = hazelcastClient;
-    var map;
-    hazelcastClient.getMap('map').then(function (mp) {
-      map = mp;
+};
 
-      map.put('key', 'value').then(function () {
-          return map.get('key');
-      }).then((res) => {
-          if(res === 'value')
-          {
-              console.log("Connection Successful!");
-              console.log("Now the map named 'map' will be filled with random entries.");
-
-              async.whilst(() => {
-                return true;
-              },(next) => {
-                var randomKey = Math.floor(Math.random() * 100000);
-                map.put('key' + randomKey, 'value' + randomKey).then(function () {
-                    map.get('key' + randomKey);
-                    if (randomKey % 100 == 0) {
-                        map.size().then((size) => console.log(`map size: ${size}`));
-                    }
-                    next();
-                });
-              },(err) => {
-                client.shutdown();
-              });
-          }
-          else {
-              throw new Error("Connection failed, check your configuration.");
-          }
-      });
-    });
-});
\ No newline at end of file
+(async () => {
+    try {
+        const client = await Client.newHazelcastClient(clientConfig);
+        const map = await client.getMap('map');
+        await map.put("key", "value")
```

Comment:
nit: it's better to keep string literals consistent by using `'` instead of `"`. Thus, `await map.put("key", "value")` becomes `await map.put('key', 'value')`. The same consideration applies to the below lines.